lib/secp256k1/schnorr.ex

```diff
@@ -0,0 +1,85 @@
+defmodule Bitcoinex.Secp256k1.Schnorr do
+  @moduledoc """
+  Schnorr-specific secp256k1 operations
+  """
+  alias Bitcoinex.Secp256k1
+  alias Bitcoinex.Secp256k1.{Math, Params, Point, PrivateKey, Signature}
+  alias Bitcoinex.Utils
+
+  @n Params.curve().n
+
+  @generator_point %Point{
```

Comment:
maybe we can move generator_point under Secp256k1 or Secp256k1.Params(expose it as public function) so it can be shared in both Schnorr and Ecdsa mod?

Reply:
I did this initially but unfortunately that creates a circular dependency between params and point modules